README.md

```diff
@@ -95,7 +95,7 @@ GoogleTagManager.registerFunctionCallTagHandler(
     // functionName is passed for convenience. In this example it will be equal to "some_function".
     // tagArguments is an object and is populated based on Tag configuration in TagManager interface.
     console.log("Handling Function Call tag:", functionName);
- }
+  }
```

Comment:
Preferably no whitespace changes like these

Reply:
I wanted it to match the function indentation (4 lines up).